difficulties.lua

```diff
@@ -24,6 +43,19 @@ function HandleDifficultyCommand ( Split, Player )
 	if (Split[2] == "peaceful") or (Split[2] == "0") or (Split[2] == "p") then
 		SetWorldDifficulty(Player:GetWorld(), 0)
 		SendMessage( Player, "World difficulty set to peaceful" )
+		
+		--Remove mobs which are not allowed in peaceful
+		Player:GetWorld():ForEachEntity(
+			function (a_Entity)
+				if not(a_Entity:IsMob()) then
+					return
+				end
+				local Monster = tolua.cast(a_Entity, "cMonster")
+				if (Monster:GetMobFamily() == cMonster.mfHostile) then
+					Monster:Destroy()
+				end
+			end
+			)

```

Comment:
No problem. I appreciate that you take the time to correct me. 
I want to contribute to Cuberite more and want to do it right in order to learn as much as possible in the process.
I'll change it.